src/top_model/main.cpp

```diff
@@ -33,184 +33,215 @@ using TIME = NDTime;
 
 
 /***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct inp_control : public cadmium::in_port<Message_t>{};
-struct inp_1 : public cadmium::in_port<Message_t>{};
-struct inp_2 : public cadmium::in_port<Message_t>{};
+struct inp_control : public cadmium::in_port<message_t> {};
+struct inp_1 : public cadmium::in_port<message_t> {};
+struct inp_2 : public cadmium::in_port<message_t> {};
 /***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct outp_ack : public cadmium::out_port<Message_t>{};
-struct outp_1 : public cadmium::out_port<Message_t>{};
-struct outp_2 : public cadmium::out_port<Message_t>{};
-struct outp_pack : public cadmium::out_port<Message_t>{};
+struct outp_ack : public cadmium::out_port<message_t> {};
+struct outp_1 : public cadmium::out_port<message_t> {};
+struct outp_2 : public cadmium::out_port<message_t> {};
+struct outp_pack : public cadmium::out_port<message_t> {};
 
 
 /********************************************/
 /****** APPLICATION GENERATOR *******************/
 /********************************************/
 template<typename T>
-class ApplicationGen : public iestream_input<Message_t,T> {
-public:
-  ApplicationGen() = default;
-  ApplicationGen(const char* file_path) : iestream_input<Message_t,T>(file_path) {}
+class ApplicationGen : public iestream_input<message_t,T>{
+    public:
+    ApplicationGen() = default;
+    ApplicationGen(const char* file_path) : iestream_input<message_t,T>(file_path) {}
 };
 
 
-int main(int argc, char ** argv) {
+int main(int argc, char ** argv){
 
-  if (argc < 2) {
-   cout << "you are using this program with wrong parameters. Te program should be invoked as follow:";
-   cout << argv[0] << " path to the input file " << endl;
-   return 1; 
-  }
+    if (argc < 2){
+        cout << "you are using this program with wrong parameters.The program should be invoked as follow:";

```

Comment:
Line is too long. You can use this
cout << "you are using this program with wrong parameters."
                "The program should be invoked as follow:";

Reply:
resolved as per suggestion